web-console/4.0-designs/role-based-access-control/role-based-access-control.md

```diff
@@ -0,0 +1,34 @@
+# Role Based Access Control (RBAC)
+
+## Object Lists
+
+![no access to a list in a single projects](img/list-single.png)
+- If a user does not have permissions to view a list of objects in a single namespace, they should be presented with a message that reads `You do not have permission to view <object name> in <project name and badge>` and should be prompted to contact the owner of the project to request access.
+- Below the message, the page should display the exact wording of the error message in a code format.
+- It is possible that a user may still have permission to create objects, and the create button should be enabled in this case.
+
+![no access to a list in all projects](img/list-all.png)
+- If a user does not have permissions to view a list of objects across namespaces, the message should read `You do not have permission to view <object name> across All Projects` and users should be prompted to contact their cluster administrator rather than the project owner.
+
+![no permission to create objects in a single namespace](img/create-single.png)
+- If a user has permission to view a list of objects in a single namespace but not create new objects, the create button should be disabled with the message `Insufficient permissions to create <resource name> in this project`.
+- Even if a user does not have Permission to create objects at the cluster scope, the create button should still be enabled. The process should succeed if they enter a namespace in which they have permission and fail if they do not have adequate permissions.
+
+
+
+## Object Details
+
+![no access to view the details of an object](img/details.png)
```

Comment:
Should the Actions dropdown be disabled for this case?

Reply:
Weirdly, it's possible to be able to edit something without being able to view it. I left this here to show that option.